src/platform_impl/windows/window_state.rs

```diff
@@ -219,11 +223,17 @@ impl WindowFlags {
 
   pub fn to_window_styles(self) -> (WINDOW_STYLE, WINDOW_EX_STYLE) {
     let (mut style, mut style_ex) = (Default::default(), Default::default());
-    style |= WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_SYSMENU | WS_CAPTION | WS_MINIMIZEBOX;
+    style |= WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_SYSMENU | WS_CAPTION;
     style_ex |= WS_EX_ACCEPTFILES;
 
     if self.contains(WindowFlags::RESIZABLE) {
-      style |= WS_SIZEBOX | WS_MAXIMIZEBOX;
+      style |= WS_SIZEBOX;
+    }
+    if self.contains(WindowFlags::RESIZABLE | WindowFlags::MAXIMIZABLE) {
```

Comment:
I don't think this is right – or at any rate, it's a change in behaviour from before. Previously, if `resizable` was disabled, the maximize button was also disabled. (On other platforms this is handled automatically by the window manager, but not on Windows). I have maintained that previous behaviour.

With your change, if `resazable` is disabled the mazimize button will still work unless `maximizable` is also disabled.
Is that what you intended?

Reply:
yeah, previously, I coupled both together because we didn't have the option to disable maximize alone but now that we do, it is fine to separate them

Reply:
Okay, yeah, I can see an argument both ways but I agree with keeping the existing behaviour for now.
(Also I guess changing it would technically be a breaking change – so `minor` instead of `patch`)